web/gallery-pearl.html

```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE html>
```

Comment:
Good idea, did that. The Gallery UI with code links is not really great, though.